File: 30 Days of Code/07 - Day 6 - Let's Review.py
# ========================
#       Information
# ========================

# Direct Link: https://www.hackerrank.com/challenges/30-review-loop/problem
# Difficulty: Easy
# Max Score: 30
# Language: Python

# ========================
#         Solution
# ========================

N = int(input())

for _ in range(N):
    listChars = [char for char in input()]
    word_1 = ""
    word_2 = ""
    check = True

    for i in range(len(listChars)):
        if check:
            word_1 += listChars[i]
        else:
            word_2 += listChars[i]
        check = not check

    print(word_1, word_2)

File: 30 Days of Code/C#/07 - Day 6 - Let's Review.cs
// ========================
//       Information
// ========================

// Direct Link: https://www.hackerrank.com/challenges/30-review-loop/problem
// Difficulty: Easy
// Max Score: 30
// Language: C#

// ========================
//         Solution
// ========================

using System;

class Solution {
	static void Main(String[] args) {
		int counter = Int32.Parse(Console.ReadLine());

		for (int i = 1; i <= counter; i++) {
			string myString = Console.ReadLine();

			for (int j = 0; j < myString.Length; j++) {
				if (j % 2 == 0) {
					Console.Write(myString[j]);
				}
			}

			Console.Write(" ");

			for (int j = 0; j < myString.Length; j++) {
				if (j % 2 == 1) {
					Console.Write(myString[j]);
				}
			}
			Console.WriteLine();
		}
	}
}
